src/users/views/table_view.py

```diff
@@ -52,7 +52,7 @@ def all_instances(self):
         """
         Retrieve all instances of the model.
         """
-        return User.objects.all().order_by("id")
+        return User.objects.all().order_by("id")[::-1]
```

Comment:
@loststoner You can use `("-id")` here, instead of `[::-1]`, to reverse the order.

More info: https://www.w3schools.com/django/django_queryset_orderby.php